### ISSUE TITLE
tag_serial_number as parameter for get_animals()?

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn#184]
@PieterjanVerhelst @jreubens Is it common to have a list of `tag_serial_numbers` that you then want the animals for? If so, it might be worth it to add `tag_serial_number` as a parameter to `get_animals()`:

```R
get_animals(tag_serial_number = c("A", "B"))
```

**Comment by PieterjanVerhelst** [comment]
Yes! I now do this through subsetting (`filter()`). Does such a functionality already exist for the `tag_id_code` as well?

**Comment by jreubens** [comment]
would definitely help to easily have access to tag serial numbers in  `get_animal`
